src/main/features/ccj/views/check-and-send.njk

```diff
@@ -54,11 +53,11 @@
       %}
     {% endif %}
 
-    {{ rowWithMultipleValue("When you want the defendant to pay", paymentOptionDisplayValue, changeLink = paymentOptionUrl) }}
+    {{ rowWithMultipleValue('When you want the defendant to pay', paymentOptionDisplayValue, changeLink = paymentOptionUrl) }}
     {{ tableEnd() }}
 
     <div class="form-group">
-      {{ statementOfTruth(form, title = 'Declaration', warningMessage = "Once you 'Sign and submit', you cannot change your answers.") }}
+      {{ statementOfTruth(form, title = 'Declaration', warningMessage = 'Once you \'Sign and submit\', you cannot change your answers.') }}
```

Comment:
I thought we used double quotes so we wouldn't have to escape characters?

Reply:
I think IDEA does this automatically sometimes :(

Reply:
Nope, that was me :). AFAIK we decided to use single quotes and `’` for words like "don’t", "you’re"

@geek-soft